aries-site/src/pages/templates/user-feedback-collection.mdx

```diff
@@ -0,0 +1,142 @@
+| **3**  | **Footer **  | Contains two actions: "Submit" and "Dismiss" if used in Layer modal. Otherwise only "Submit" is needed inline. | <Checkmark a11yTitle="true" size="small" /> |  Submit button sends survey responses to the HPE customer experience team and initiates thank you message. If presented in a modal, the Dismiss button allows the user to bypass the survey and continue their product experience. |
+
+## Implementation
+
+### Feedback question styling
+Feedback surveys are composed with Grommet Form and FormFields containing any input types. Add the 
+`kind="survey"` prop to Form to apply special survey styling to FormField labels and inputs.
+
+```
+<Form 
+  kind="survey" // Apply feedback styling to form field labels and inputs
+  ... 
+>
+  <FormField 
+    htmlFor="question-1" 
+    name="QID1" 
+    label="How was the experience deploying your workload?"
+  >
+    <ThumbsRating id="question-1" name="QID1" />
+  </FormField>
+</Form>
+```
+
+### Feedback setup and submission
+To initiate user feedback collection, please contact the <a href={`mailto:${contact.email}?subject=Collecting%20user%20feedback%20in%20a%20product%20experience`}>unified customer experience team</a> for the following steps:
+
+1. A dedicated survey will be created for your use case. At that time, an API token, survey id, and response id will be provided.
+1. Obtain the survey response schema with the [retrieve a survey response call](https://api.qualtrics.com/1179a68b7183c-retrieve-a-survey-response). The properties in the "values" object represent the data which should be submitted with each new survey response.
+1. Survey responses are submitted using [create a new response](https://api.qualtrics.com/f1cad92018d2b-create-a-new-response).
+
+```
+// Submit survey response
+fetch(`https://yul1.qualtrics.com/API/v3/surveys/${SURVEY_ID}/responses`, {
+  method: 'POST',
+  headers: {
+    'Content-Type': 'application/json',
+    'X-API-TOKEN': API_TOKEN,
+  },
+  body: JSON.stringify(data),
+})
+.then(response => {
+  // Response handling here
+})
+.catch(error => {
+  // Error handling here
+});
+```
+
+## Examples
+
+### Solicited feedback
+Solicited feedback is an active method of asking users for their opinions and perspectives. 
+A feedback questionnaire is presented after a user has completed a task or consumed a piece of 
+content. Solicited feedback should not interrupt or impede a user from completing their intended 
+objective.
+
+#### Task flow in center layer
+
+Click on "Power 1 device" to see the feedback questions presented at the completion of the 
```

Comment:
Replace with "Click on "Power on 1 device" since the button has the word "on" in it too.

Reply:
changed